android/java/org/chromium/chrome/browser/crypto_wallet/util/JavaUtils.java

```diff
@@ -70,4 +77,29 @@ public static boolean anyNull(Object... items) {
         }
         return false;
     }
+
+    @SafeVarargs
+    public static <T> T[] asArray(T... items) {
+        return items;
+    }
+
+    /**
+     * Returns a combined string with a separator or an empty string. Empty and null values are
+     * ignored.
+     * @param separator to append after each string. Pass null for no separator
+     * @param items of string values.
+     * @return a combined or empty string.
+     */
+    public static String concatStrings(char separator, String... items) {
+        if (items == null) return "";
+        StringBuilder builder = new StringBuilder();
+        for (int i = 0; i < items.length; i++) {
+            String item = items[i];
+            if (TextUtils.isEmpty(item)) continue;
+            builder.append(item).append(separator);
+        }
+        // Delete separator after the last value
+        builder.deleteCharAt(builder.length() - 1);
+        return builder.toString();
```

Comment:
I wanna delete the separator at last, and your code doesn't check for nulls or empty. 
BTW, the focus is not on refactoring things to streams. Probably just use it where it is more suitable but should not be enforced.

Reply:
I don't understand your answer.
I didn't add a `filter` to the chain as I just wanted to suggest a more elegant way, and give you the idea. Not sure about you mention the focus on refactoring the code using streams.

Reply:
I have started using streams because now we can https://bravesoftware.slack.com/archives/C03MMJ4CHEZ/p1675178531068709 but this does not mean this we should use this everywhere possible to replace the existing working solution. Plus I also wanted to remove the char at last and the above solution is easy and more efficient than streams.

It's also good practice to not recommend partial or incomplete solutions without information. I think it is better to add subjective suggestions as nit and would recommend the same to separate important vs subjective feedbacks.

Reply:
Thank you for the feedback and further elaboration @Pavneet-Sing. 
I completely agree with what you said about the use of Java streams.

It's important to remember that we're all human, and suggestions may not always be perfect, but that should not deter us from constructive criticism. It provides invaluable insights! 

Code reviews have many purposes, not only merging the code back to the main branch. They are an invaluable opportunity to discuss many things, just to name a few:
- Provide feedback and suggestions
- Creating connections with other team members
- Raising awareness of current coding styles and best practices

All of this becomes extremely hard (or impossible) if feedback is not accepted gracefully or (even worse) when the answers become rude. Demonstrating a courteous and receptive attitude towards feedback is crucial for a productive and harmonious workplace.
Really! I couldn't stress this point enough, we should try to be excellent to each other, and being courteous with those who review your code is the foundation for a fully functional team. 

Let's continue with this offline, feedback is an essential part of the development process, it plays a crucial part, but discussing it in PR comments may not be appropriate.
It's better to continue these conversations offline to prevent clogging up the PR comments section. Once again, thanks for your valuable input!

Reply:
I didn't deny the feedback or anything that you've mentioned above with a long paragraph and details, which often not needed. been seeing this patterns many times. The extra details was provided because it was asked.
I just clearly explained that we should not be tempted to use streams and should not enforce this in review, especially with partial solutions and full understanding. Would suggest to the same to take [things more gracefully](https://github.com/brave/brave-core/pull/17615#discussion_r1138442359).